### DISCUSSION TITLE
how to convert html+latex into docx

### DISCUSSION THREAD

**Discussion by todaysprint** [jgm/pandoc#9476] [Q&A] [answered]
hello
my Html file contains matha latex code too 
example  $7\frac{3}{5} \times 10 + 2\frac{4}{5} \times 30 = ? \times 8$	
how to convert along with html file

**Comment by jgm** [comment, marked as answer]
`-f html+tex_math_dollars`
(or one of the other `tex_math_*` extensions, depending on how the math appears in your HTML file)

**Reply by todaysprint** [reply]
thank you for quick response
